source/SpinalHDL/Semantic/when_switch.rst

```diff
@@ -50,7 +50,7 @@ It is possible to define new signals inside a when/switch statement:
      val tmp = a + b
      x := tmp
      y := tmp + 1
-   } otherwise {
+   } .otherwise {
```

Comment:
I am not sure of what you mean: it is now valid Spinal code without the dot (it doesn’t work for me). Or the dot has been added in another PR ?

Reply:
Hmm i just tested on scala 2.11.12 spinalhdl dev, it works without the dot for me. 
Which version of scala / SpinalHDL did you used ?

Reply:
Tested again with scala 2.13.6. It works only if otherwise is on the same line as the closing bracket.
```scala
} otherwise { // works
} .otherwise { // works
```
```scala
}
otherwise { // don’t work
```

```scala
}
.otherwise { // works
```
If you suggest that the dot-less notation works, you have to include **big** warnings in the docs that it must be on the same line. Or people (like me) will get burnt with this subtlety.

Reply:
+1 for the dot less, with a warning in the doc ^^